pkg/config/manager.go

```diff
@@ -30,6 +30,12 @@ type Manager struct {
 	dir       string
 }
 
+// ErrConfigNotFound means that the manager cannot find a config using a name/id.
+var ErrConfigNotFound = errors.New("no match found")
+
+// ErrTooManyConfigs means that there are more than one config with a given name.
```

Comment:
This is not necessarily `more than one config with a given name`:

`ErrTooManyConfigs means that there are more than one config results for a given search.`